README.md

```diff
@@ -296,9 +296,12 @@ In the example above, the compiler would search for a module interface file firs
 
 #### `--base-library=DIR`, `-b DIR`
 
-Predefined data types and operations are not build directly into the compiler but are part of the *base library* that accompanies the compiler.
-The compiler uses the same mechanism that is used to load module dependencies to load modules from the base library, i.e., module interface files.
-In contrast to automatically generated module interface files, the base library does not use the JSON file format but TOML since the module interfaces of the base library are maintained manually and TOML is a more user friendly format.
+Predefined data types and operations are not built directly into the compiler but are part of the *base library* that accompanies the compiler.
+
+To load module dependencies from the base library, i.e. module interface files, the compiler uses the same mechanism that is used to load module dependencies.
```

Comment:
I think `i.e. module interface files` was meant to explain the `mechanism` and not  `module dependencies`.

```suggestion
To load module dependencies from the base library, the compiler uses the same mechanism, i.e. module interface files, that is used to load module dependencies.
```